src/audio/volume/volume.c

```diff
@@ -347,7 +347,8 @@ static inline void volume_ramp(struct processing_module *mod)
 		}
 	}
 
-	set_volume_process(cd, dev, true);
+	if (cd->is_passthrough)
```

Comment:
Hmm, better to say 1 MCPS, 6 MCPS is an optimistic value. The peak MCPS varies a lot and would need a lot of runs and statistic to claim this.